game/dancegraph.cc

```diff
@@ -114,7 +114,7 @@ void DanceGraph::setupJoinMenu() {
 		}
 		m_selectedTrack = ConfigItem(ol); // Create a ConfigItem from the option list
 		m_selectedTrack.select(cur); // Set the selection to current level
-		m_menu.add(MenuOption("", _("Select track")).changer(m_selectedTrack)); // MenuOption that cycles the options
+		m_menu.add(MenuOption{"", _("Select track")}).changer(m_selectedTrack); // MenuOption that cycles the options
```

Comment:
not a big fan of initializing with {} (especially when the members are not specified  (designators are C++20: https://en.cppreference.com/w/cpp/language/aggregate_initialization).